inc/customizer.php

```diff
@@ -14,10 +14,39 @@ function _s_customize_register( $wp_customize ) {
 	$wp_customize->get_setting( 'blogname' )->transport         = 'postMessage';
 	$wp_customize->get_setting( 'blogdescription' )->transport  = 'postMessage';
 	$wp_customize->get_setting( 'header_textcolor' )->transport = 'postMessage';
+
+	if ( isset( $wp_customize->selective_refresh ) ) {
+		$wp_customize->selective_refresh->add_partial( 'blogname', array(
+			'selector'        => '.site-title a',
+			'render_callback' => '_s_customize_partial_blogname',
```

Comment:
Looking at the documentation it would be 

```php
function() {
      return get_bloginfo( 'name', 'display' );
}
```

https://make.wordpress.org/core/2016/02/16/selective-refresh-in-the-customizer/